lib/internal/fs/watchers.js

```diff
@@ -133,7 +133,7 @@ function FSWatcher() {
       if (this._handle !== null) {
         // We don't use this.close() here to avoid firing the close event.
         this._handle.close();
-        this._handle = null;  // Make the handle garbage collectable
+        this._handle = null;  // Make the handle is garbage collectable.
```

Comment:
```suggestion
// Make the handle garbage collectable.
```

Reply:
:smile: was probably thinking of the other sentence while changing this. thanks.